depthai_ros_driver/CMakeLists.txt

```diff
@@ -82,6 +85,10 @@ add_library(
 
 ament_target_dependencies(${COMMON_LIB_NAME} ${COMMON_DEPS})
 
+# Link OpenCV to the common lib so cv headers/libs are available
+target_link_libraries(${COMMON_LIB_NAME} ${OpenCV_LIBS})
```

Comment:
You can link in the same line, although I believe we don't have to link OpenCV specifically if we already link `cv_bridge` (OpenCV functions are already used in `segmentation.cpp`)